tests/integration/actions/templar/base.py

```diff
@@ -50,13 +50,17 @@ class BaseClass:
     @staticmethod
     @pytest.fixture(scope="module", name="tmux_session")
     def fixture_tmux_session(request):
-        """tmux fixture for this module"""
+        """tmux fixture for this module
```

Comment:
```suggestion
        """Return a new tmux session.
```
(this is also what's displayed in the output of `pytest --fixtures`)

Reply:
Sorry, out of scope here. Linting and doc string will slowly be fixed in other PRs.

Reply:
(ignoring as out of the scope)